### ISSUE TITLE
Planetwars: multiple moves

### ISSUE THREAD

**Issue by iasoon** [ZeusWPI/MOZAIC#58]
I believe it would simplify bot implementation (especially blockly style bots) if we allowed multiple moves per turn.

Opinions?

**Comment by wschella** [comment]
More flexibility without overhead would indeed result in easier bot implementation I guess. So I agree that we want to change to this. 

Though I propose to block changing this until we integrate everything. And changes can clearly become features instead fixes.

**Comment by iasoon** [comment]
We introduced this limitation to make the game simpler, and to disable strong players to totally outplay beginners by using simultaneous moves. On a second thought, I think this restriction achieves the opposite: advanced strategies are still possible via carefully planning your moves ahead, letting expeditions piggyback on other expeditions (so that you only need one command instead of two) and other way harder techniques. Allowing the simple alternative of multiple moves each turn, we will even the playing field a bit.

Therefore, I believe the current implementation of our intention could be marked a 'bug' ;-)

**Comment by wschella** [comment]
Agreed. I feel me might be able to use the version we have now as 'harder' game-mode someday.
